docs/user_guide.rst

```diff
@@ -28,60 +28,72 @@ After installation Qiskit on IQM can be imported in your Python code as follows:
 Running a quantum circuit on an IQM quantum computer
 ----------------------------------------------------
 
-In this section we construct a simple quantum circuit and demonstrate how to execute it on an IQM quantum computer.
+In this section we demonstrate the practicalities of using Qiskit on IQM on an example of constructing and executing
+a simple quantum circuit on an IQM quantum computer.
 
 .. note::
 
    At the moment IQM does not provide a quantum computing service open to the general public.
    Please contact our `sales team <https://www.meetiqm.com/contact/>`_ to set up your access to an IQM quantum computer.
 
 
-Let's consider the following quantum circuit which prepares and measures a Bell state:
+Let's consider the following quantum circuit which prepares and measures a GHZ state:
 
 .. code-block:: python
 
     from qiskit import QuantumCircuit
 
-    qc = QuantumCircuit(2, 2)
+    qc = QuantumCircuit(3, 3)
     qc.h(0)
     qc.cx(0, 1)
-    qc.measure([0, 1], [0, 1])
+    qc.cx(0, 2)
+    qc.measure_all()
 
     print(qc.draw(output='text'))
 
 ::
 
-         ┌───┐     ┌─┐
-    q_0: ┤ H ├──■──┤M├───
-         └───┘┌─┴─┐└╥┘┌─┐
-    q_1: ─────┤ X ├─╫─┤M├
-              └───┘ ║ └╥┘
-    c: 2/═══════════╩══╩═
-                    0  1
-
-First, we need to decompose it into IQM's native gate family:
+            ┌───┐           ░ ┌─┐
+       q_0: ┤ H ├──■────■───░─┤M├──────
+            └───┘┌─┴─┐  │   ░ └╥┘┌─┐
+       q_1: ─────┤ X ├──┼───░──╫─┤M├───
+                 └───┘┌─┴─┐ ░  ║ └╥┘┌─┐
+       q_2: ──────────┤ X ├─░──╫──╫─┤M├
+                      └───┘ ░  ║  ║ └╥┘
+    meas_0: ═══════════════════╩══╬══╬═
+                                  ║  ║
+    meas_1: ══════════════════════╩══╬═
+                                     ║
+    meas_2: ═════════════════════════╩═
+
+To execute this circuit against an IQM quantum computer you need to initialize an appropriate Qiskit backend instance
+that represents the IQM quantum computer under use, and simply use Qiskit's ``execute`` function as normal:
 
 .. code-block:: python
 
-    from qiskit.compiler import transpile
+    from qiskit import execute
+    from qiskit_iqm import IQMProvider
 
-    qc_decomposed = transpile(qc, basis_gates=['r', 'cz'])
+    provider = IQMProvider(iqm_server_url)
+    backend = provider.get_backend()
 
-    print(qc_decomposed.draw(output='text'))
+    job = execute(qc, backend, shots=1000)
```

Comment:
This currently usually does not work, and one has to use `optimization_level=0` to make it work. This shall be fixed by https://github.com/Qiskit/qiskit-terra/issues/8970